apps/crn-server/src/routes/analytics.route.ts

```diff
@@ -64,7 +64,7 @@ export const analyticsRouteFactory = (
   );
   analyticsRoutes.get(
     '/collaboration/team',
-    async (req, res: Response<ListTeamCollaborationResponse>) => {
+    async (req, res: Response<ListTeamCollaborationAlgoliaResponse>) => {
```

Comment:
Do we want to call the type  "Algolia" even in the `crn-server`? As in `ListTeamCollaborationAlgoliaResponse` instead of `ListTeamCollaborationResponse` even if ListTeamCollaborationResponse = ListTeamCollaborationAlgoliaResponse. 

I know it's very minor but just wanted to make sure that's the idea 😅

Reply:
we dont, that's a good point. These should be two different types - I changed the Response type in the model without looking where it was being imported. To be fair this bit got me confused:
https://github.com/yldio/asap-hub/blob/master/packages/model/src/common.ts#L4
someone thought adding these algolia specific properties to this type was a good idea. I will split these